worker/services/hub_service.py

```diff
@@ -0,0 +1,18 @@
+import httpx
+import os
+
+def call_hub_post(endpoint, data, headers={}):
+  response = httpx.post(os.environ['HUB_URL'] + endpoint, json=data, headers=headers, timeout=600)
```

Comment:
i think we need to have a deafult on this to be the hub url we want, because while building the executable i think it will try to get from this env and it will not exists on the PC of the guy running it